community/kernel/src/test/java/org/neo4j/test/OnDemandJobScheduler.java

```diff
@@ -94,7 +94,10 @@ public Runnable getJob()
 
     public void runJob()
     {
-        job.run();
+        if ( job != null )
```

Comment:
If there are no jobs, we do nothing. I don't see any problem. This is test code anyway.